data/bunrodea/bunrodea ships.txt

```diff
@@ -0,0 +1,496 @@
+# Copyright (c) 2021 by Amazinite
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+ship "Kaiken"
+	sprite "ship/kaiken"
+	thumbnail "thumbnail/kaiken"
+	attributes
+		category "Fighter"
+		"cost" 1199000
+		"shields" 3200
+		"hull" 800
+		"required crew" 1
+		"bunks" 1
+		"mass" 50
+		"drag" 1
+		"heat dissipation" 0.95
+		"outfit space" 121
+		"weapon capacity" 35
+		"engine capacity" 45
+		weapon
+			"blast radius" 40
+			"shield damage" 400
+			"hull damage" 200
+			"hit force" 600
+	outfits
+		"Locust Blaster" 3
+		
+		"Solar Battery"
+		"Solar Cell" 27
+		"Small Shield Relay"
+		"Shield Relay Limiter" 6
+		
+		"Chiisana Rift Thruster" 2
+		"Chiisana Rift Steering"
+	
+	engine -5.5 30.5
+	engine 5.5 30.5
+	gun -7 -31
+	gun 7 -31
+	gun 0 -20
+	leak "flame" 60 80
+	explode "tiny explosion" 20
+	explode "small explosion" 15
+	description "While not particularly fast by fighter standards, where the Kaiken lacks in speed it makes up for in durability and firepower; it boasts a significant amount of weapon space and ports for up to three guns. Should the Kaiken need to retreat, each fighter has a built in cloaking device and a small amount of fuel that should be enough to allow the fighter to return to its carrier for repairs."
```

Comment:
They don't seem to have any fuel.

Reply:
The cloak was removed at some point, and the fuel removed with it.